### ISSUE TITLE
Video stream latency of 2 secs

### ISSUE THREAD

**Issue by manjunathshetty** [arut/nginx-rtmp-module#945]
Hi,

We are using nginx-rtmp-module to stream live camera feed, we are seeing about 2 secs of delay.

We convert RTSP to RTMP with ffmpeg command
`ffmpeg -i rtsp://10.71.11.115:8554/stream -c copy -f flv rtmp://localhost:1935/nas/stream`

We are not doing any decoding in ffmpeg, does ffmpeg still induce delay ?
or it is the rtmp module that induce delay ?

Thanks

**Comment by PaulBGD** [comment]
Yeah I'm seeing about 2s as well. Might be network latency?

**Comment by manjunathshetty** [comment]
@PaulBGD  No, for me its not the network latency for sure because i am using local network with less than 100ms latency. I am suspecting it may be latency induced by ffmpeg or the Nginx-rtmp-module itself

**Comment by PaulBGD** [comment]
@manjunathshetty For me it turned out to be OBS's buffer.

**Comment by Prendo93** [comment]
@manjunathshetty try setting the ffmpeg -probesize to around 200000. 
This is the number of bytes ffmpeg waits for to determine the stream information
